src/lib/tabs/tab-truncate.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({ name: 'mdTruncate' })
+export class MdTabLabelTruncate implements PipeTransform {
+  transform(value: string, limit: number = 70, trail: String = '…'): string {
```

Comment:
Thinking about this more in depth, I think that overall this approach has too many i18n complications. 
* Splitting on whitespace is fine for English, but not as good for, say, German ([longer words](https://www.youtube.com/watch?v=cfzcJj4-JuU)) or Chinese (no spaces). 
* Splitting words could change the label's meaning (more of a problem for locales other than English)

Reply:
I'm more thinking we shelve this until next quarter and I can consult with our internal i18n experts

Reply:
No worries, should I make PR for the `max-width: 264px;` css I added here to make it adhere to the spec?

Reply:
If we're not going to do the truncation I'd rather put off changing the `max-width` as well- that'll be a breaking change that would be time consuming to deal with now